site/OWNERS

```diff
@@ -2,3 +2,5 @@
 
 approvers:
 - moficodes
+reviewers:
```

Comment:
```suggestion

reviewers:
```

nits: Let's clarify the block as root OWNERS https://github.com/kubernetes-sigs/kueue/blob/6e8d4143e4969d424597d9a30c7672f6ea1210fc/OWNERS

Reply:
I don't know what your comment means.

Reply:
@tenzen-y can you clarify?

Reply:
I meant adding a new line between the `approvers` and `reviewers` sections.
However, it seems to be resolved, already.